interface-definitions/load-balancing-inbound-ipvsadm.xml.in

```diff
@@ -0,0 +1,227 @@
+              <help>Linux Virtual Server</help>
+            </properties>
+            <children>
+              <tagNode name="rule">
+                <properties>
+                  <help>Backend server load balancing rules</help>
+                  <valueHelp>
+                    <format>u32:0-65535</format>
+                    <description>Number for this rule</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 0-65535"/>
+                  </constraint>
+                  <constraintErrorMessage>Back-end server load balancing rule numbers are not between 0 and 65535</constraintErrorMessage>
+                </properties>
+                <children>
+                  <leafNode name="disable">
+                    <properties>
+                      <help>Disable load balancing rules</help>
+                      <valueless/>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="protocol">
+                    <properties>
+                      <help>Load balancing protocol</help>
+                      <completionHelp>
+                        <list>tcp udp sctp</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>tcp</format>
+                        <description>TCP Protocol</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>udp</format>
+                        <description>UDP Protocol</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>sctp</format>
+                        <description>SCTP Protocol</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>^(tcp|udp|sctp)$</regex>
+                      </constraint>
+                      <constraintErrorMessage>The load balancing protocol must be one of tcp, udp, and sctp</constraintErrorMessage>
+                    </properties>
+                    <defaultValue>tcp</defaultValue>
+                  </leafNode>
+                  <leafNode name="virtual-ip">
+                    <properties>
+                      <help>Virtual server IP</help>
+                      <valueHelp>
+                        <format>ipv4</format>
+                        <description>IPv4 VIP for the virtual server</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ipv6</format>
+                        <description>IPv6 VIP for the virtual server</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="ip-address"/>
+                      </constraint>
+                    </properties>
+                    </leafNode>
+                    <leafNode name="virtual-port">
```

Comment:
Any reason to not use our generic "port" XML include? Why use a new node?

Reply:
done